{{ cookiecutter.project_slug }}/{{ cookiecutter.project_slug }}/settings/email.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+EMAIL_BACKEND = os.getenv(
+    'EMAIL_BACKEND', 'django.core.mail.backends.smtp.EmailBackend'
+)
+
+EMAIL_USE_TLS = os.getenv('EMAIL_USE_TLS', 'False') == 'True'
```

Comment:
Please refer do django-environ docs, as they have native support for bool's that's better than manual comparisons we used in GAM.

Reply:
...also for database URLs, as it seems!